addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
I am not sure if I fully understood the problem. But if the device only supports toggle and there is no way to tell in which state it is, I would also go for implementing a toggle channel, although this isn't ideal either.
From the architectural view, it is no issue to add "software functionality" into a binding, i.e. abstracting some functionality that the raw hardware does not provide. If this only works when the user knows that he has to set up his device in a certain way (i.e. configuring his scenes in RFXCOM in a special way), this is indeed a bit nasty and should rather be avoided.